src/components/RadioButton/__stories__/radioButton.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import RadioButton from "../RadioButton";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { TOGGLE, CHECKBOX, DROPDAWN } from "../../../storybook/components/related-components/component-description-map";
+import DialogContentContainer from "../../DialogContentContainer/DialogContentContainer";
+import { Link } from "../../../storybook/components";
+import "./radioButton.stories.scss";
+
+<Meta
+  title="Components/Radio Button"
+  component={ RadioButton }
+/>
+
+<!--- Component template -->
+
+export const radioButtonTemplate = createComponentTemplate(RadioButton);
+
+<!--- Component documentation -->
+
+# RadioButton
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+A radio represents an item in a single selection list.
+<Canvas>
+  <Story name="Overview"
+         args={ {text: "Selection", checked: true } }>
+    { radioButtonTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ RadioButton } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "A radio lets a user make exactly one selection from a set, which is all visible at the same time.",
+  "Set one radio group option as default. Select the most likely or convenient option.",
+  "Ensure both label and input are clickable to select the option.",
+  "Place radio buttons  can be placed vertically or horizontaly, using 16px spacing.",
+]}/>
+
+<Tip>When there’s limited space or no default selection, consider using a select <Link href="/?path=/docs/components-dropdown--overview" size={Link.sizes.SMALL} withoutSpacing>Dropdown</Link> instead.</Tip>
+
+## Variants
+### States
+Radio buttons have 3 states: regular, selected, and disabled.
+<Canvas>
+  <Story name="States">
+    <RadioButton text="Regular" />
+    <RadioButton text="Selected" checked />
+    <RadioButton text="Disabled" disabled />
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+          <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" />
+            <RadioButton text="Item 2" checked />
+          </DialogContentContainer>,
+        description:"Use radio buttons when only one item can be selected from a list."
+      },
+      negative: {
+        component:
+          <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" checked />
+            <RadioButton text="Item 2" checked />
+          </DialogContentContainer>,
+        description:<>Don't use <Link href="/?path=/docs/components-checkbox--overview" withoutSpacing>checkboxes</Link> when only one item can be selected from a list. Use radio buttons instead.</>
```

Comment:
I think this row belong to checkbox and not radiobutton. I saw the same mistake in the figma, please talk with Rotem what to write here insted